src/Sidenav/context.ts

```diff
@@ -39,17 +39,24 @@ export const propsDeepSearch = (
       }
     } else {
       for (const prop in theObject) {
-        if (theObject.hasOwnProperty(prop)) {
-          if (prop === targetProp) {
-            if (theObject[prop] === targetValue) {
-              finalResults.push(theObject);
+        if (
+          prop === '$$typeof' ||
+          prop === 'children' ||
+          prop === 'props' ||
+          prop === targetProp
+        ) {
+          if (theObject.hasOwnProperty(prop)) {
+            if (prop === targetProp) {
```

Comment:
lines 49 and 50 could be joined with `&&` since there is no else for the first condition.

Reply:
Done